services/app-api/src/postgres/user/updateCmsUserProperties.ts

```diff
@@ -46,9 +72,27 @@ export async function updateCmsUserProperties(
                     },
                 },
             }),
+            client.userAudit.create({
```

Comment:
You could probably do this in a single .update() command by using `updatedByAuditEntries: { create: {...}` if you wanted to get rid of the transaction here. It's a 6 of one situation though, I don't see strong reasons to prefer one over the other.

Reply:
I like the `transaction` as a signal that these _need_ to happen together.